modules/xcode/xcode_common.lua

```diff
@@ -987,12 +987,32 @@
 			settings['COPY_PHASE_STRIP'] = 'NO'
 		end
 
-		settings['GCC_C_LANGUAGE_STANDARD'] = 'gnu99'
+		if (p.languages.isc(cfg.language)) then
+			if cfg.language ~= "C" then
+				settings['GCC_C_LANGUAGE_STANDARD'] = cfg.language:lower()
```

Comment:
Based on previous comment, this will need to handle things like "C++Latest" (but a C equivalent), so it's probably best this entire block for adding `GCC_C_LANGUAGE_STANDARD` be a function instead, same for the C++ block.